man/generate-manpages.sh

```diff
@@ -51,3 +51,13 @@ argparse-manpage \
     --project-name "modulemd-generate-macros" \
     --url HOMEPAGE \
     > ./man/modulemd-generate-macros.1
+
+
+argparse-manpage \
+    --pyfile bld2repo/cli.py \
+    --function get_arg_parser \
+    --author "Martin Čurlej" \
+    --author-email "mcurlej@redhat.com" \
+    --project-name "bld2repo" \
+    --url HOMEPAGE \
+    > ./man/bld2repo.1
```

Comment:
And here is one missing newline at the end of file. Your IDE is behaving very strangely IMHO. Basically, the last line in the file should have `\n` at its end (then this warning symbol won't appear in the Github UI) but there shouldn't be an empty line at the end (you won't see a line that is empty but has a line number in the diff).

I don't need it in this PR, just saying so you can fix your IDE settings once you have the time.